wremnants/unfolding_tools.py

```diff
@@ -93,8 +93,8 @@ def define_gen_level(df, gen_level, dataset_name, mode="wmass"):
 
     return df
 
-def select_fiducial_space(df, accept=True, mode="wmass", pt_min=None, pt_max=None, mass_min=60, mass_max=120, mtw_min=0, selections=[]):
-    # Define a fiducial phase space and either select events inside/outside
+def select_fiducial_space(df, select=True, accept=True, mode="wmass", pt_min=None, pt_max=None, mass_min=60, mass_max=120, mtw_min=0, selections=[]):
```

Comment:
What is exactly the interplay between accept and select? It seems select=False removes the cut on the acceptance, is this meant to get an inclusive histogram?